model/cluster_installation_migration_database.go

```diff
@@ -0,0 +1,27 @@
+package model
+
+import (
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	// DatabaseMigrationStatusSetupIP indicates that database migration setup is still running.
+	DatabaseMigrationStatusSetupIP = "setup-in-progress"
+	// DatabaseMigrationStatusSetupComplete indicates that database migration setup is completed.
+	DatabaseMigrationStatusSetupComplete = "setup-complete"
+	// DatabaseMigrationStatusTeardownIP indicates that database migration teardown is still running.
+	DatabaseMigrationStatusTeardownIP = "teardown-ip"
```

Comment:
this should probably be `teardown-in-progress` to match `replication-in-progress` and `setup-in-progress`

Reply:
> Yeah, first glance at the code looks pretty good, but it's easier to track requests for changes (if there are any) by not squashing/rebasing after reviews start. It's covered here: https://mattermost.com/blog/submitting-great-prs/

Thanks for the explanation. Unfortunately, It will be a bit of a pain to rebase the commits in this PR. Is that ok if I do that for the upcoming ones?